DigitalLearningSolutions.Web/ViewModels/TrackingSystem/CourseSetup/CourseDetails/EditAutoRefreshOptionsViewModel.cs

```diff
@@ -0,0 +1,42 @@
+namespace DigitalLearningSolutions.Web.ViewModels.TrackingSystem.CourseSetup.CourseDetails
+{
+    using System.ComponentModel.DataAnnotations;
+    using DigitalLearningSolutions.Data.Models.Courses;
+    using DigitalLearningSolutions.Web.Attributes;
+
+    public class EditAutoRefreshOptionsViewModel
+    {
+        public EditAutoRefreshOptionsViewModel() { }
+
+        public EditAutoRefreshOptionsViewModel(CourseDetails courseDetails)
+        {
+            CustomisationId = courseDetails.CustomisationId;
+            RefreshToCustomisationId = courseDetails.RefreshToCustomisationId;
+            AutoRefreshMonths = courseDetails.AutoRefreshMonths.ToString();
+            ApplyLpDefaultsToSelfEnrol = courseDetails.ApplyLpDefaultsToSelfEnrol;
+        }
+
+        public EditAutoRefreshOptionsViewModel(
+            int customisationId,
+            int? refreshToCustomisationId,
+            string? autoRefreshMonths,
```

Comment:
I think there was a validation or accessibility reason it was better to be a string (can't remember exactly what)

Reply:
Ah ok fair enough.
I don't know if it's worth raising this separately. Although I don't think it would be much of a concern or priority. 
I would be curious to look at it more.